services/workflows-service/prisma/migrations/20240418113024_removed_apm_from_payment_method/migration.sql

```diff
@@ -0,0 +1,14 @@
+/*
+  Warnings:
+
+  - The values [apn] on the enum `PaymentMethod` will be removed. If these variants are still used in the database, this will fail.
+
+*/
+-- AlterEnum
+BEGIN;
+CREATE TYPE "PaymentMethod_new" AS ENUM ('credit_card', 'debit_card', 'bank_transfer', 'pay_pal', 'apple_pay', 'google_pay');
+ALTER TABLE "TransactionRecord" ALTER COLUMN "paymentMethod" TYPE "PaymentMethod_new" USING ("paymentMethod"::text::"PaymentMethod_new");
+ALTER TYPE "PaymentMethod" RENAME TO "PaymentMethod_old";
+ALTER TYPE "PaymentMethod_new" RENAME TO "PaymentMethod";
+DROP TYPE "PaymentMethod_old";
+COMMIT;
```

Comment:
It won't fail on dev/sb/prod? I think there are existing transactions with the APN payment method.

Reply:
Yeah I was about to comment on that. If we want to remove the `apn` value, we need to change existing data in all environments, something like:
`UPDATE "TransactionRecord" SET "paymentMethod"='apple_pay' WHERE "paymentMethod"='apn'`

Reply:
Can you add it to your migration?